src/main/java/io/github/koxx12_dev/skyclient_installer_java/MainCode.java

```diff
@@ -95,10 +95,12 @@ public void code(List<String> mods, List<String> packs, String mc) throws IOExce
             String url;
             String file = (String) modsjson.getJSONObject(i).get("file");
 
-            if (id.equals("rpm")) {
-                url = "https://github.com/nacrt/SkyblockClient-REPO/raw/main/files/mods/Resource_Pack_Manager_1.2.jar";
-            } else {
-                url = (String) modsjson.getJSONObject(i).get("url");
+            boolean has_url = false;
+            try {
+                url = "https://github.com/nacrt/SkyblockClient-REPO/raw/main/files/mods/" + file
+                url = (String) modsjson.getJSONObject(i).get("url")
+            }
+            catch (Exception) {
```

Comment:
you cant leave Expection empty it need to save to some var so for example 
`Exception ignored` will work but just `Exception` wont